CRM/Member/Form/MembershipRenewal.php

```diff
@@ -553,6 +553,8 @@ protected function submit() {
         $paymentParams = array_merge($paymentParams, $contributionRecurParams);
       }
 
+      $paymentParams['invoiceID'] = $payment_params['invoice_id'];
```

Comment:
should the latter one be `$paymentParams`

Reply:
Aye. D'oh.